### DISCUSSION TITLE
convention used in the code for cam_pose_cw/cam_pose_wc

### DISCUSSION THREAD

**Discussion by sumitsarkar1** [stella-cv/stella_vslam#105] [Q&A] [answered]
Just curious to know whats the convention : cam_pose_cw means pose of the Camera w.rt. .to World coordinate system or vice versa

**Comment by ymd-stella** [comment, marked as answer]
`cam_pose_cw` is a mapping ![texclip20210617074614](https://user-images.githubusercontent.com/7959916/122306671-d9470200-cf43-11eb-80bb-4e791a45b8e1.png) that maps a point in the world coordinate to a point in the camera coordinate.

> pose of the Camera w.rt. .to World coordinate system

It depends on the definition of Pose, but in general the inverse of `cam_pose_cw` (`cam_pose_wc`) is "pose of the Camera w.rt. .to World coordinate system".
